packages/cdktf-cli/lib/models/terraform-cloud.ts

```diff
@@ -116,7 +116,9 @@ export class TerraformCloud implements Terraform {
   constructor(
     public readonly stack: SynthesizedStack,
     public readonly config: TerraformJsonConfigBackendRemote,
-    isSpeculative = false
+    isSpeculative = false,
+    // TODO: find out how to forward logs from terraform cloud (only public to ignore ts error)
```

Comment:
todo left -> follow up issue?

Reply:
Nope, big chunk of work missing from this PR 🙈